Source/Core/DolphinNoGUI/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+if(NOT(USE_X11 OR ENABLE_HEADLESS))
```

Comment:
Yeah, `nogui` means we don't have the usual wX gui interface, not that the game doesn't render in a  window. Headless implies nogui, but not the other way around. On my original fork I changed the executable to `dolphin-emu-headless` if headless was on.